### ISSUE TITLE
Use location instead of organization

### ISSUE THREAD

**Issue by citizenrich** [synthetichealth/synthea#334]
I would like to use Synthea to generate data that references Location resources (for facilities) rather than Organization resources. Thoughts on what a PR would require to support it and if the maintainers would accept it?

**Comment by jawalonoski** [comment]
Hmm... well, a straight-forward PR would require changing four different exporters: `fhir` (STU3), `fhir_dstu2` (DSTU2), `hospital.fhir` (STU3), and `hospital.fhir_dstu2` (DSTU2).

Those exporters can be found in `src/main/java/org/mitre/synthea/export`.

However, I don't think we'd accept a PR that just made a one-for-one swap... well, maybe we would (I'm open to persuasion), but that would might just need to be changed again. Perhaps where we want to go is to have both `Organization` and `Location` together once we add clinicians as `Practitioner` resources and we can generate synthetic data suitable for a Provider Directory, e.g. http://www.fhir.org/guides/argonaut/pd/index.html

@citizenrich what is your use-case for `Location`? Provider Directory? What advantage do you see `Location` having over `Organization`?

I'd be interested in having others comment on this issue. Adding clinicians/practitioners, and locations could be a big chunk of work.

**Comment by citizenrich** [comment]
Indeed, thanks for the discussion. It would seem like quite a bit of work. I'm happy to have it discussed and learn what you all think.

The use case is to try to match what we (a global health NGO) see in resource-constrained countries and Location resources are more accurate for such contexts. So, imagine creating a synthetic dataset that includes immunizations for vaccine-preventable diseases in Liberia where most facilities are government-led and small. Hospitals as Organizations aren't a terribly accurate representation. But, I could be wrong.

**Comment by citizenrich** [comment]
Just updating this issue to note that there's now an international locations repository to support Synthea:
https://github.com/synthetichealth/synthea-international
